cmd/cmd_test.go

```diff
@@ -33,7 +37,11 @@ func TestEnvVar(t *testing.T) {
 
 	require.NoError(t, cmd.Execute())
 	assert.Equal(t, `export TF_VAR_availability_zone_names='["us-west-1a"]'
-export TF_VAR_docker_ports='[{ external = 8300, internal = 8300, protocol = "tcp" }]'
+export TF_VAR_docker_ports='[{
+  external = 8300
+  internal = 8300
+  protocol = "tcp"
+}]'
```

Comment:
Thank you so much for the PR.

After doing a bit of testing, I found that the export statement above will not be properly parsed by Terraform. I have to admit that the current implementation of the export statement is a bit hacky: https://github.com/shihanng/tfvar/blob/c118267763107f9e094524649f3253e756341202/pkg/tfvar/tfvar.go#L53-L71

I am trying to look for a better approach before merging this PR. If you have any suggestion, please feel free to share it here. Thank you.

Reply:
I guess the simplest solution to sort of rollback to the old behavior would be to add the following after line 62

```go
b = bytes.ReplaceAll(b, []byte("\n"), []byte(``))
```

Reply:
It looks like this is the relevant change in hclwrite, which breaks our env var use case: https://github.com/hashicorp/hcl/commit/926e53e3381e8460d1b31cbd6a6fbbd51a7b3d7a

To me, it looks like we need to add the old version of `appendTokensForValue` to `tfvar` in order to generate the old one line format.

Reply:
Hi @breml, thank you so much for looking into this :heart: . 

Adding the old version of `appendTokensForValue` seems to work! However, I want to avoid maintaining an older version of `appendTokensForValue` if possible.

Based on your findings in https://github.com/shihanng/tfvar/pull/3#discussion_r482719709, I found out that newlines only occur inside map / object types. Which means we can kind of replace the newlines with commas surgically without complicated logics.

Please let me know what you think about this commit: 9498702f023235e1ba0ec98d9a58fe291ab9fb69. If you agree I want to include this into your PR/branch and add another commit to remove `generate.go` file.

Thank you.

Reply:
Hi @shihanng 

I fully understand, that you want to avoid maintaining an older version of `appendTokensForValue`.

I had a brief look at your proposal/commit and the solution you found looks good to me. The reason I did not go down this route was, that it is easily possible, that they will make further changes to the standard way of formatting HCL in the future (e.g. add newlines to lists as well) and `tfvar` would then always need to adopt to these changes.
But I guess, this is the minor evil.

Therefore, feel free to go ahead with your proposal.

Reply:
> that they will make further changes to the standard way of formatting HCL in the future (e.g. add newlines to lists as well)

Yes, you are right. If things become too complicated. We might need to come back with the approach you propose in cbab11b. 

> Therefore, feel free to go ahead with your proposal.

Thank you. I will include the changes and release the new version.